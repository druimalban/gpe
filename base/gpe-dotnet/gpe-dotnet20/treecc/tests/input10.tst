// literal code sections mixed with non-literal blocks
{ abc^*&%/* kjk}jkj*/
	// }
	/*
*/
	"\"}" '/*kjkjk*/'
}
no_parse_literal
{ parse_literal abc // }
}
{ hello }
