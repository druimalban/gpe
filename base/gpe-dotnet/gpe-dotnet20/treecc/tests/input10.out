line 2: literal code:
 abc^*&%/* kjk}jkj*/
	// }
	/*
*/
	"\"}" '/*kjkjk*/'

line 8: identifier (len = 16): no_parse_literal
line 9: {
line 9: identifier (len = 13): parse_literal
line 9: identifier (len = 3): abc
line 10: }
line 11: literal code:
 hello 
